docs/samples/client/oauth/csharp/UnbrandedCredential.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Demonstrates creating a client with an unbranded authentication token provider.
+using System;
+using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+
+namespace UnbrandedTypeSpec
```

Comment:
We shouldn't duplicate this. We should instead link to it.

Reply:
We can put a short sample containing just the constructor in an md file and then link to the cs file to see the full details.